bin/resty

```diff
@@ -213,6 +222,7 @@ for my $var (sort keys %ENV) {
 
 my $main_include_directives = resolve_includes('main', @main_includes);
 my $http_include_directives = resolve_includes('http', @http_includes);
+my $lua_shared_dicts = format_directives('lua_shared_dict', @shdicts);
```

Comment:
I'd rather ask before instead of implementing something and having to iterate over it again because you had something else in mind :)